website/src/pages/initiatives.js

```diff
@@ -72,30 +91,26 @@ const Community = () => {
         </section>
         <section id="meetings">
           <h3 className={styles.communitySubtitle}>Our Meetings</h3>
-          <div className={styles.communityRow}>
-            <div className={styles.communityMeeting}>
-              <h5 className={styles.meetingTitle}>Past Meetings</h5>
-              <List
-                height={500}
-                itemCount={pastMeetings.length}
-                itemSize={100}
-                width="90%"
-                itemData={{ isCompleted: true }}
-              >
-                {MeetingCard}
-              </List>
-            </div>
-            <div className={styles.communityMeeting}>
-              <h5 className={styles.meetingTitle}>Upcoming Meetings</h5>
-              <List
-                height={500}
-                itemCount={upcomingMeetings.length}
-                itemSize={100}
-                width="90%"
-                itemData={{ isCompleted: false }}
-              >
-                {MeetingCard}
-              </List>
+          <div>
+            <div>
+              <div className={styles.addToCalendar}>
+                <a href="https://workspace.google.com/intl/en_in/products/calendar/?utm_source=google&utm_medium=cpc&utm_campaign=1011352-Workspace-APAC-IN-en-BKWS-EXA-Golden&utm_content=text-ad-none-none-DEV_c-CRE_469259700126-ADGP_Hybrid%20%7C%20AW%20SEM%20%7C%20BKWS%20~%20EXA%20%7C%20Calendar%20%7C%20%5BM:1%5D%20%7C%20IN%20%7C%20EN-KWID_43700057535047290-kwd-1844967558&userloc_1007820-network_g&utm_term=KW_google%20calendar&gclid=CjwKCAiAz--OBhBIEiwAG1rIOkbt8UI2XpXuSfpJzoHLn12sVQllOHJEgzju9YKwJOe1AgCPEgUKMBoCnK8QAvD_BwE&gclsrc=aw.ds">
```

Comment:
Need to replace this google calendar link for Moja Global. We can export this static data from the `websiteData/meetings.js` folder.